tests/test_core.py

```diff
@@ -3786,9 +3792,9 @@ def test_dylink_funcpointers_wrapper(self):
       #include "header.h"
       int main(int argc, char **argv) {
         charfunc f1 = emscripten_run_script;
-        f1("out('one')");
+        f1("console.log('one')");
         charfunc f2 = get();
-        f2("out('two')");
+        f2("console.log('two')");
```

Comment:
`out` should still work, why avoid it?

Reply:
`out` gets mangled by closure, but this string here is just a normal C string which gets passed to `eval` so there is no way closure can see it (is there?)

Reply:
Oh, this is a string that gets evalled? Then sgtm.

Reply:
Yes these two strings are passed to `emscripten_run_script` which just evals the string.